config.json

```diff
@@ -57,6 +57,18 @@
         "strings"
       ]
     },
+    {
+      "slug": "isogram",
+      "uuid": "3c1b70fa-aaa8-4323-9aab-4c76da0a89e1",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "strings",
+        "algorithms",
+        "logic"
```

Comment:
I think configlet will want the topics to be sorted alphabetically

Reply:
Alphabetized